src/main/scala/persistentscheduler/persistence/InMemorySchedulerPersistence.scala

```diff
@@ -43,7 +43,7 @@ class InMemorySchedulerPersistence(initialEvents: Seq[TimedEvent] = Seq()) exten
     }
   }
 
-  override def find(eventType: String, reference: String): List[TimedEvent] = {
+  override def find(eventType: String, reference: String): util.List[TimedEvent] = {
```

Comment:
Please decide if you want to reference util.List with the package name or if you rename the import to JList